modules/querying/pages/func/vector-functions.adoc

```diff
@@ -0,0 +1,1007 @@
+= Vector Functions
+:experimental:
+
+Vector Functions in TigerGraph are currently implemented as UDFs and preloaded as part of the Graph Data Science Library:
+xref:3.9@graph-ml:similarity-algorithms:index.adoc[]
+
+
+== Vector
```

Comment:
This page is in the GSQL **Functions** documentation. It is not for **algorithms**, so do not copy the descriptions of similarity algorithms here.

My original suggestion was not to document the similarity functions here, but to direct the reader to the GDS Similarity Algorithms page, and to put the documentation there.

Reply:
I understand now. I will leave these here though. I don't think it is a bad idea to have just the functions listed here.